src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
```

Comment:
According to the BEM methodology, it's a good practice to avoid using margins for BEM blocks. It's better to use margins for BEM elements. Consider moving margins from `.card__img` to `.card`.